deploy_ansible.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+cd /home/waiter/waiter
+
+# Change to main branch if needed
+BRANCH="$(git branch --show-current)"
+if [ "$BRANCH" != "main" ]
+then
+  git checkout main -q
+fi
+
+# Deploy if changes have been made
+GIT_STATUS="$(git status -s)"
+if [ -n "$GIT_STATUS" ]
+then
+    source .venv/bin/activate
+    ansible-galaxy collection install -r requirements.yml
+    ansible-playbook playbook.yaml
+    docker compose pull
+    docker compose up --detach --remove-orphans
```

Comment:
I'm not sure you want to do this.  This will break vscode dev containers which we likely want people to be using.

Reply:
Which part of this will break vscode dev containers?  Is it sufficient to remove `--remove-orphans`?

Reply:
It is sufficient to remove `--remove-orphans` to support dev containers.